src/components/Pagination.js

```diff
@@ -1,14 +1,21 @@
 import React from 'react';
 import { BiLeftArrow, BiRightArrow } from 'react-icons/bi';
 import { PaginationContent, ButtonNumber } from './Pagination.style';
-
+import { Link } from 'react-router-dom';
 export default function Pagination({ current, totalPages }) {
+  let pages = [];
+  for (let page = 1; page <= totalPages; page++) {
+    pages.push(page);
+  }
```

Comment:
Estas líneas pueden reducirse a lo siguiente, sino me falla la memoria: 
```suggestion
  let pages = [...Array(totalPages).keys()]
```

Reply:
quedó así: 
let pages = [...Array(totalPages).keys()].map((num, index) => index + 1);